device/mellanox/x86_64-mlnx_msn4800-r0/ACS-MSN4800/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,112 @@
+{% set default_cable = '5m' %}
```

Comment:
normal case you should link to the file of 4700, unless you have some other reason.